_pages/documentation/general_docs/developement/release_procedures.md

```diff
@@ -113,20 +113,20 @@ Currently hosted in [`util/dockerfiles`](https://github.com/gem5/gem5/tree/stabl
 These images are mostly used for testing purposes.
 The [`ubuntu-20.04_all-dependencies`](https://github.com/gem5/gem5/tree/stable/util/dockerfiles/ubuntu-20.04_all-dependencies/) Dockerfile is the one most suitable for users who wish to build and execute gem5 in a supported environment.
 
-We provide pre-built Docker images, hosted at <gcr.io/gem5-test>.
+We provide pre-built Docker images, hosted at <ghcr.io/gem5>.
```

Comment:
This link doesn't go anywhere.

```suggestion
We provide pre-built Docker images, hosted at ghcr.io/gem5.
```

Reply:
I made some changes so that we don't use that link.